Hopsan-release-notes.txt

```diff
@@ -26,15 +31,77 @@
     Version History
 ******************************
 
-2.9.0 (2018-??-??)
-- Enhancement: It is now possible to choose what ports to log when simulation using HopsanCLI.
-               Use the --logonly command line argument, and specify a comma-separated list of full port names.
-               It is also possible to alternatively specify a text file with one full port name per line.
-               Examples: Gain#out or SubsystemName#Gain#out
-- Enhancement: When saving results from a HopsanCLI simulation, interface ports (system ports) will also be saved
-- Enhancement: Results saved by HopsanCLI now use scientific notation, this may increase precision for small values
-
-------------------------------
+2.9.0 (2018-06-??)
+
+--- Important Changes ---
+
+- The HopsanGenerator library refactoring. GUI parts were split into a separate library and the library now has a C API
+- All Hopsan source files now use the file suffix '.cpp' rather then mixing '.cpp' and '.cc'.
+- Update Qt to 5.6.3 on Windows (official Windows 10 support).
+- Update the 64-bit Windows compiler to MinGW 4.9.4 (from 4.9.3). If you have manually set up a build environment for
+  custom component libraries you are encouraged to update (use one of the Hopsan downloads with compiler included).
+- All binaries now use lower-case names (hopsancli.exe instead of HopsanCLI.exe).
+- Continuous integration (Travis CI) now builds builds and tests every change to the Hopsan code. This will reduce the
+  risk of new code breaking old functionality.
+
+--- Enhancements ---
+
+- Optimization:
+  - Improved script file handling.
+  - Improved variable names in plots.
+  - Points are now sorted by objective value with DE algorithm.
+  - Script generation is now shown in a separate wizard.
+  - Reduced terminal output during optimization.
+  - New algorithm, Genetic Algorithm.
+
+- FMU:
+  - Exported Hopsan FMUs will now report Warning and Error messages. Info messages will be reported as Debug type.
+  - Imported FMUs will now have their messages redirected to the Hopsan core logger instead of standard output.
+    messages should now be visible inside HopsanGUI. Enable Debug messages to see FMU Info messages.
+
+- HopsanGUI:
```

Comment:
There are also some new HCOM functions which should be mentioned (ceil, floor, round and unco). Also possibility to filter parameters by values with dipa command.

Reply:
There were some issues that were not set to mileatone 2.9.0, so I did not see then. I will add this information.